src/middleware/proxy.js

```diff
@@ -43,14 +43,19 @@ module.exports = function (target, options) {
 
   proxy.on('error', function (err, req, res) {
     var msg = err.toString() + ': ' + host + req.url;
-    log(msg);
+    logger(msg);
 
     res.writeHead(500, { 'Content-Type': 'text/plain' });
     res.end(msg);
   });
 
   proxy.on('proxyRes', function (proxyRes, req, res) {
-    log(req.url.replace(path, '') + ' -> ' + host + req.url);
+    var from = req.url;
+    if (req.url !== path) {
+      from = req.url.replace(path, '');
+    }
+
```

Comment:
please, move this changes to another PR so we can discuss there how is it working and why this changes are needed